pkg/config/config.go

```diff
@@ -449,6 +495,17 @@ func (m *Manager) setDefaults(prefix string, flags *pflag.FlagSet, config interf
 				m.viper.SetDefault(name, val)
 				flags.DurationP(name, shorthand, val, description)
 
+			case *time.Duration:
+				var def string
+				if val != nil {
+					def = fmt.Sprintf("%v", *val)
+					m.viper.SetDefault(name, def)
+				}
+				flags.StringP(name, shorthand, def, description)
+
+			case *tls.Config:
+				// TODO: Support SMTP TLS config https://github.com/TheThingsNetwork/lorawan-stack/issues/2925
```

Comment:
Not here. See `pkg/config/tlsconfig` for how we handle TLS.

Reply:
Maybe add a comment about this?